doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "NASYS LoRaWAN® Outdoor Gateway"
+description: ""
+weight: 1
+---
+
+Technical information regarding the LoRaWAN® Outdoor Gateway can be found in [the official product page][1]. This page guides you to connect it to {{% tts %}}.
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways.
+2. LoRaWAN® Outdoor Gateway connected to the Internet (or your local network) via Ethernet.
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}). Typically, the **EUI** field for your gateway should exist on the sticker at the bottom.
+
+## Configuration using a Terminal
+
+Find the IP address the gateway. This can be done in various ways. You can connect your machine to the same local network as that of the gateway Ethernet connection and scan for open SSH ports or assign a static IP to the gateway and use that. Once the gateway IP address is found, ssh into it.
+
+```bash
+$ ssh root@<GatewayIP>
+```
+
+The default username is **root**, and the default password can be also found in the sticker. It is typically suggested that you change the default password using `passwd` after logging in.
+
+Your gateway should come with a slightly modified version of the [Lora-net UDP packet forwarder](https://github.com/Lora-net/packet_forwarder) pre-installed at `/opt/nas-lgw`. There are two configuration files `global_conf.json` and `local_conf.json`, both located in `/opt/nas_lgw`.
+
+Typically, you should not need to touch the `global_conf.json` file. Instead, edit `local_conf.json`
+
+```bash
+$ cp /opt/nas-lgw/local_conf.json /opt/nas-lgw/local_conf.json.bak
+$ vi /opt/nas-lgw/local_conf.json
+```
+
+Edit the server parameters.
+
+1. **gateway_ID**: Set to the gateway EUI (in lowercase).
+2. **server_address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) this is the same as what you use instead of `https://thethings.example.com`.
+3. **serv_port_up**: UDP upstream port of the Gateway Server, typically 1700.
+4. **serv_port_down**: UDP downstream port of the Gateway Server, typically 1700.
+
+You can also update the rest of the parameters, as you see fit. For example, update the **ref_latitude**, **ref_longitude** and **ref_altitude** parameters to update the Fake GPS location reported by the gateway in its status messages.
+
+Save the configuration and restart the packet forwarder:
+
+```bash
+$ systemctl restart nas_lgw
+```
+
+If your configuration was successful, your gateway will connect to {{% tts %}} after a couple of seconds. You can easily check if your gateway has connected using the Console.
```

Comment:
> You can easily check if your gateway has connected using the Console.

We don't write this in the Tektelic guide. Let's keep them consistent (i.e. either add it there, or remove it here)

Reply:
removed it from here as well